web/ui/templates/status.html

```diff
@@ -54,7 +54,8 @@ <h2 id="alertmanagers">Alertmanagers</h2>
         </tr>
         {{range .Alertmanagers}}
         <tr>
-          <td>{{.}}</td>
+          {{/* Alertmanager URLs always have Scheme, Host and Path set */}}
+          <td>{{.Scheme}}://<a href="http://{{.Host}}">{{.Host}}</a>{{.Path}}</td>
```

Comment:
Currently the only real alternative is https though and this might end up breaking for users that are not forwarding http to https.